server_structs/director.go

```diff
@@ -78,6 +78,21 @@ type (
 		DirlistHost   string       `json:"dirlisthost"`
 	}
 
+	// Downtime represents a server downtime event
+	Class    string
+	Severity string
+	Downtime struct {
+		UUID        string   `json:"uuid" gorm:"primaryKey"`
```

Comment:
```suggestion
		UUID        string   `json:"id" gorm:"primaryKey"`
```

Sorry, this one is a bit of a knit pick but we use `id` in the api token and the namespace endpoints it is best to just keep things consistent and use id here.